documentation/modules/cruise-control/ref-cruise-control-configuration.adoc

```diff
@@ -40,6 +40,30 @@ spec:
     # ...
 ----
 
+[[cors-configuration]]
+== Cross-Origin Resource Sharing configuration
+
```

Comment:
_Add a general definition of CORS in a separate paragraph:_

Cross-Origin Resource Sharing (CORS) allows you to specify allowed methods and originating URLs for accessing REST APIs.